bibtexbrowser.php

```diff
@@ -1194,13 +1194,13 @@ function getBibLink($iconurl=NULL) {
   }
 
   /** returns a "[pdf]" link if relevant. modified to exploit the new method, while keeping backward compatibility */
```

Comment:
I am not the original author of the remark "modified to exploit the new method, while keeping backward compatibility", but I have changed the documentation, I hope this is what you wanted.